tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
     semantic_version>=2.6.0
     argparse>=1.4.0
     argcomplete>=1.6.0
-    ruamel.yaml>=0.14.2
+    ruamel.yaml==0.14.12
```

Comment:
I suppose it's better to use `ruamel.yaml >=0.14.0, <0.15.0` because this allows for bug fix releases (I hope I used the right syntax).

Reply:
ok, i will check and fix that

Reply:
Same should be done in `setup.py` and `requirements.txt`, no? BTW: Do we need all three of them? Seems kind of redundant to me. tox seems to support including `requirements.txt` since `v1.6.1` (v3.0.0rc1 is the most recent). So I assume it's a feature safe to use. No idea about `setup.py`.

Reply:
ok, i will check

Reply:
BTW: The PR does `import json`. But it's not part of `requirements.txt`.
```
$ find tools -name "*.py" | xargs cat | grep import | grep -v "^#" | sort | uniq
    from helpers import *
    from hilbert_cli_config import *
    from hilbert_config.helpers import *
    from hilbert_config.hilbert_cli_config import *
    from hilbert_config.subcmdparser import *
    from subcmdparser import *
    import dill
    import platform
    import ruamel.yaml as yaml
    import semantic_version
# import traceback
from __future__ import absolute_import, print_function, unicode_literals
from os import path
import argparse  # NOQA
import logging
import os
import sys

$ find tests -name "*.py" | xargs cat | grep import | grep -v "^#" | sort | uniq
from __future__ import absolute_import, print_function, unicode_literals  # NOQA
from helpers import *
from hilbert import print_query_result
from hilbert_cli_config import *
from hilbert_cli_config import SemanticVersionValidator, load_yaml
from os import path
from ruamel.yaml.compat import PY2, PY3, text_type, string_types, ordereddict
from subcmdparser import *
import logging
import os                            # NOQA
import pytest                        # NOQA
import semantic_version              # NOQA
import sys
```

Reply:
AFAIR `json` is part of the standard library starting with Python 2.6: https://github.com/python/cpython/tree/2.7/Lib/json